content/en/docs/configuration/auto-recover-childcluster-changes.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Recover All Changes Happens in Child Clusters."
+description: "This helps rollback unexpected operations (like deleting, updating) that occurred solely inside a child cluster."
+date: 2023-03-21
+draft: false
+weight: 3
+---
+
+> Clusternet-hub and agent provide a feature gate `Recovery` which allows to rollback any changes inside a child cluster.
+> It's `false` by default.
+
+So after you try [Deploying Applications to Multiple Clusters with Replication Scheduling](../../tutorials/multi-cluster-apps/replication-scheduling-to-multiple-clusters/),
+in child cluster `child-1`, and scale the replicas of `nginx-deploy` from 3 to 1, the replicas remains at 1:
+```shell
+root@child-1:~# kubectl  scale deploy my-nginx -n foo --replicas=1
+root@child-1:~# kubectl  get po -n foo -w
+NAME                        READY   STATUS    RESTARTS   AGE
+my-nginx-66b6c48dd5-9dpxt   1/1     Running   0          58s
+```
+As has described the feature gate `Recovery` is `false` by default, so if you want to ensure the resources deployed 
+by Clusternet exist persistently in a child cluster, you should set this feature gate to `true`:``--feature-gates=Recovery=true``
```

Comment:
Please add a state that this feature gate is set on `clusternet-agent`.

Reply:
> clusternet

Yes, i missed the important thing, please take another look.